dashboard/test/ui/features/foundations/hamburger.feature

```diff
@@ -1,13 +1,104 @@
-@no_mobile
 @single_session
 Feature: Hamburger dropdown
 
+  @eyes_mobile
+  Scenario: Signed-out user sees the hamburger on mobile
+    Given I am on "http://code.org/"
+    And I open my eyes to test "Hamburger Menu in English Signed-out"
+    And I rotate to portrait
+    And I wait for 0.5 seconds
+    Then I wait until element "#hamburger-icon" is visible
+    And I scroll the "#hamburger-icon" element into view
+    And I click selector "#hamburger-icon"
+    And I see no difference for "Hamburger Menu: English Signed-out"
+    And I close my eyes
+
+  @eyes_mobile
+  Scenario: Signed-out user sees Teach, About and Legal dropdowns in hamburger
+    Given I am on "http://code.org/"
+    And I open my eyes to test "Hamburger Dropdowns in English Signed-out"
+    And I rotate to portrait
+    And I wait for 0.5 seconds
+    Then I wait until element "#hamburger-icon" is visible
+    And I scroll the "#hamburger-icon" element into view
+    And I click selector "#hamburger-icon"
+    Then I wait until element "#educate_entries" is visible
+    And I click selector "#educate_entries"
+    Then I scroll the "#educate-overview" element into view
+    And I see no difference for "Teach Entries Mobile Part 1 Hamburger Menu: English Signed-out"
+    Then I scroll the "a:contains(Online Community)" element into view
+    And I see no difference for "Teach Entries Mobile Part 2 Hamburger Menu: English Signed-out"
+    Then I wait until element "#educate_entries" is visible
+    And I click selector "#educate_entries"
+    Then I wait until element "#about_entries" is visible
+    And I click selector "#about_entries"
+    Then I scroll the "#about_entries-items" element into view
+    And I see no difference for "About Entries Hamburger Menu: English Signed-out"
+    And I click selector "#about_entries"
+    Then I wait until element "#legal_entries" is visible
+    And I click selector "#legal_entries" if it exists
+    Then I scroll the "#legal_entries-items" element into view
+    And I see no difference for "Legal Entries Hamburger Menu: English Signed-out"
+    And I close my eyes
+
+  @eyes
+  Scenario: Student user sees the hamburger dropdown
```

Comment:
Same for this test. I think its steps are repeated in the next test so I think you could remove this test alltogether.